src/test/TEST_ams/solvers.saved.lassen

```diff
@@ -1,117 +1,117 @@
 # Output file: solvers.out.0
-    Cycle 15   8.419126e-05    0.462177     6.270087e-06 
-    Cycle 16   3.893120e-05    0.462414     2.899374e-06 
-    Cycle 17   1.800904e-05    0.462586     1.341211e-06 
-    Cycle 18   8.331652e-06    0.462637     6.204941e-07 
+    Cycle 15   6.551570e-05    0.464364     4.879237e-06 
+    Cycle 16   3.043337e-05    0.464520     2.266504e-06 
+    Cycle 17   1.413954e-05    0.464606     1.053032e-06 
+    Cycle 18   6.572920e-06    0.464861     4.895138e-07 
 
 
- Average Convergence Factor = 0.452014
+ Average Convergence Factor = 0.446099
 
 # Output file: solvers.out.1
-    Cycle 15   8.419126e-05    0.462177     6.270087e-06 
-    Cycle 16   3.893120e-05    0.462414     2.899374e-06 
-    Cycle 17   1.800904e-05    0.462586     1.341211e-06 
-    Cycle 18   8.331652e-06    0.462637     6.204941e-07 
+    Cycle 15   6.551570e-05    0.464364     4.879237e-06 
+    Cycle 16   3.043337e-05    0.464520     2.266504e-06 
+    Cycle 17   1.413954e-05    0.464606     1.053032e-06 
+    Cycle 18   6.572920e-06    0.464861     4.895138e-07 
 
 
- Average Convergence Factor = 0.452014
+ Average Convergence Factor = 0.446099
 
 # Output file: solvers.out.2
-    Cycle 39   3.440551e-05    0.722036     2.562327e-06 
-    Cycle 40   2.484170e-05    0.722027     1.850069e-06 
-    Cycle 41   1.793733e-05    0.722065     1.335871e-06 
-    Cycle 42   1.295116e-05    0.722022     9.645285e-07 
+    Cycle 45   2.495866e-05    0.756777     1.858779e-06 
+    Cycle 46   1.889154e-05    0.756913     1.406935e-06 
+    Cycle 47   1.430435e-05    0.757183     1.065307e-06 
+    Cycle 48   1.083145e-05    0.757214     8.066648e-07 
 
 
- Average Convergence Factor = 0.719067
+ Average Convergence Factor = 0.746545
 
 # Output file: solvers.out.3
-    Cycle 39   3.440551e-05    0.722036     2.562327e-06 
-    Cycle 40   2.484170e-05    0.722027     1.850069e-06 
-    Cycle 41   1.793733e-05    0.722065     1.335871e-06 
-    Cycle 42   1.295116e-05    0.722022     9.645285e-07 
+    Cycle 45   2.495866e-05    0.756777     1.858779e-06 
+    Cycle 46   1.889154e-05    0.756913     1.406935e-06 
+    Cycle 47   1.430435e-05    0.757183     1.065307e-06 
+    Cycle 48   1.083145e-05    0.757214     8.066648e-07 
 
 
- Average Convergence Factor = 0.719067
+ Average Convergence Factor = 0.746545
```

Comment:
It looks like problem 2 and 3 converge quite a bit slower now. Are those the same problems as before?

Reply:
I think the difference was from using PMIS coarsening on GPUs vs. HMIS on CPU.